libyul/backends/wasm/BinaryTransform.h

```diff
@@ -81,12 +107,13 @@ class BinaryTransform
 
 	static bytes encodeName(std::string const& _name);
 
-	std::map<std::string, size_t> m_locals;
-	std::map<std::string, size_t> m_globals;
-	std::map<std::string, size_t> m_functions;
-	std::map<std::string, size_t> m_functionTypes;
-	std::vector<std::string> m_labels;
-	std::map<std::string, std::pair<size_t, size_t>> m_subModulePosAndSize;
+	std::map<std::string, size_t> m_registeredLocals;
+	std::vector<std::string> m_registeredLabels;
+
+	std::map<std::string, size_t> const m_registeredGlobals;
```

Comment:
Why is it important that they are "registered"?

Reply:
I'm not too attached specifically to `registered` but disambiguating between `_globals`/`_functions` and `m_globals`/`m_functions` was the main point of this refactor :) If you get both in a function the names make it seem like one of them is redundant.

Looking at the diff again, I see that now they're never used alongside each other so I guess the name is no longer a problem. If the rest of the refactor is acceptable then I can safely drop the prefix.